File: abc/abc173/abc173_b.fs
// https://atcoder.jp/contests/abc173/submissions/15022356

let N = stdin.ReadLine() |> int

let S =
    [| for _ in [ 1..N ] do
        yield stdin.ReadLine() |]

let counter j =
    S
    |> Array.filter (fun s -> s = j)
    |> Array.length
    |> string

"AC x " + (counter "AC") |> stdout.WriteLine
"WA x " + (counter "WA") |> stdout.WriteLine
"TLE x " + (counter "TLE") |> stdout.WriteLine
"RE x " + (counter "RE") |> stdout.WriteLine

File: abc/abc173/abc173_b.cs
// https://atcoder.jp/contests/abc173/submissions/15019863

using System;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());

        var result = new int[4];
        for (int i = 0; i < N; i++)
        {
            var S = Console.ReadLine();
            switch (S)
            {
                case "AC":
                    result[0]++;
                    break;
                case "WA":
                    result[1]++;
                    break;
                case "TLE":
                    result[2]++;
                    break;
                case "RE":
                    result[3]++;
                    break;
            }
        }
        Console.WriteLine($"AC x {result[0]}");
        Console.WriteLine($"WA x {result[1]}");
        Console.WriteLine($"TLE x {result[2]}");
        Console.WriteLine($"RE x {result[3]}");
    }
}
